from0k2bp.tex

```diff
@@ -1863,7 +1863,7 @@
 receive back a challenge $x$, both sides recalculate $C'$, continue until a
 final step (each step a halving and a new $L, R$), and in the last step reveal
 scalars for the now single values $a, b$, and the Verifier makes the final
-check that $C^{*} = a^{*}b^{*}G + a^{*}G_1 + b^{*}H_1$, where * indicates the $\log_2n$-th transformed values.
+check that $C^{*} = (a^{*} \cdot b^{*})G + a^{*}G_1 + b^{*}H_1$, where * indicates the $\log_2n$-th transformed values.
```

Comment:
I believe these are now single values not vectors (as per previous line), so we don't want a dot product here. Admittedly it does look really crappy using a \* superscript here, though.

Reply:
You're right. When I was first reading this, I thought the protocol could end on vectors of length 2, for which you would need the dot product. I think the number of messages stays the same, but I think I invented that out of thin air.